timemachine/fe/single_topology.py

```diff
@@ -55,12 +59,39 @@ def recursive_map(items, mapping):
 
 
 def find_chiral_bonds(ring_bonds, proper_idxs, proper_params, mol):
-    # a stereo bond is defined as a bond that
+    """
+    Find chiral bonds - note that a chiral bond is a local descriptor. i.e. only the neighboring
+    atoms to the bond are considered. Furthermore, all atoms are considered unique, even if they are hydrogens!
+
+    (ytz): we can simplify the API to just bond_idxs, and infer ring atoms and degree from the
+    connectivity alone without resorting to rdkit.
+
+    Parameters
+    ----------
+    ring_bonds: list of 2-tuple
+        Bonds that are part of a ring system.
+
+    proper_idxs: list of 4-tuple
+        Proper torsion idxs
+
+    proper_params: list of 3-tuple
+        Proper torsion parameters
+
+    mol: Chem.Mol
+        RDKit molecule
+
+    Returns
+    -------
+    set
+        Set of chiral bonds.
+
+    """
+    # a chiral bond is defined as a bond that
     # 1) has a single proper torsion term with k > 10 kJ/mol, period=2, phase=3.1415
-    # 2) a bond that is not part of a ring system.
+    # 2) is not part of a ring system.
     # the reason why 2) is present is because the planar torsions spanning a
-    # ring system are not used to enforce stereochemistry, since if we simply
-    # disabled them, we would *still* get the correct stereochemistry due to steric effects.
+    # ring system are not used to enforce chiralchemistry, since if we simply
```

Comment:
nit: chiralchemistry -> chiral chemistry

Reply:
This came from an overly zealous stereo->chiral replacement. I'm trying to avoid the use of the words stereo/stereochemistry due to confusion with CIP, so I will rename this to "chirality".